app/views/partials/_audit_trail.erb

```diff
@@ -0,0 +1,50 @@
+  <p>
+    <strong>Audit Log:</strong>
+    <table>
+      <thead>
+        <tr>
+          <td>
+            Changed by:
+          </td>
+          <td>
+            Time of change:
+          </td>
+          <td>
+            Property Changed:
+          </td>
+          <td>
+            Changed From:
+          </td>
+          <td>
+            Changed To:
+          </td>
+        </tr>
+      </thead>
+      <tbody>
+        <% auditable.audit_trail_events.each do |event| %>
+          <% event.event_changes.keys.each do |change_key| %>
+            <% change_from, change_to = event.event_changes[change_key] %>
+            <% next if change_key == 'updated_at' %> 
+            <tr>
+              <td>
+                <%= event.user.email %>
+              </td>
+              <td title="<%= event.created_at %>">
```

Comment:
Notice the text content of the table cell is in the human friendly format of `#{n} minutes ago`, but I also put the exact timestamp in the markup for people who want to know the exact time.

This timestamp should be localised using javascript (for modern browsers using `new Date(date). toLocaleDateString()`. The same goes for the election end and start dates.